shoprunback.php

```diff
@@ -63,6 +63,11 @@ public function __construct()
             'AdminShoprunback' => ['name' => 'ShopRunBack', 'parent' => 'SELL']
         ];
 
+        \Shoprunback\RestClient::getClient()->setCustomHeaders([
+            'Prestashop-Version: ' . _PS_VERSION_,
+            'Shoprunback-Module-Version: ' . $this->version
+        ]);
```

Comment:
does the headers are set for all request done on the API ?

Reply:
Yes. Once set at the beginning, the instance will have the customHeaders set for any request done afterwards. And since when you want to do anything in a PS' module, it passes by the main module class, the headers will always be set.